tests/conftest.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+
+import os
```

Comment:
`os` is a global module and has to be listed first. You may have mixed it up with pytest.